posthog/models/feature_flag.py

```diff
@@ -311,12 +356,55 @@ def get_active_feature_flags(
 
 # Return feature flags with per-user overrides
 def get_overridden_feature_flags(
-    team_id: int, distinct_id: str, groups: Dict[GroupTypeName, str] = {},
+    team_id: int, distinct_id: str, groups: Dict[GroupTypeName, str] = {}, hash_key_override: Optional[str] = None
 ) -> Dict[str, Union[bool, str, None]]:
-    feature_flags = get_active_feature_flags(team_id, distinct_id, groups)
```

Comment:
Is this feature flag still used anywhere? If so, it's a sign that maybe it shouldn't be?

Reply:
not sure I get this. Do you mean overrides only applying to active feature flags? Or something else?

Reply:
I mean `get_active_feature_flags` as a function